graphql-dgs-codegen-core/src/main/kotlin/com/netflix/graphql/dgs/codegen/generators/java/DataTypeGenerator.kt

```diff
@@ -362,6 +368,33 @@ abstract class BaseDataTypeGenerator(
         addFieldWithGetterAndSetter(fieldDefinition.type, fieldDefinition, javaType)
     }
 
+    private fun addIsDefinedFieldWithGetters(javaType: TypeSpec.Builder) {
+        val fields = javaType.build().fieldSpecs
+        fields.forEach() {
+            val fieldName = "is${it.name.capitalized()}Defined"
+            val field = FieldSpec
+                .builder(com.squareup.javapoet.TypeName.BOOLEAN, fieldName)
+                .addModifiers(Modifier.PRIVATE)
+                .initializer("false")
+                .build()
+            val getterName = "getIs${it.name.capitalized()}Defined"
+            val getter = MethodSpec
+                .methodBuilder(getterName)
+                .addModifiers(Modifier.PUBLIC)
+                .returns(com.squareup.javapoet.TypeName.BOOLEAN)
+                .addStatement(
+                    "return \$N",
+                    ReservedKeywordSanitizer.sanitize("${generateBooleanFieldName(it.name)}")
```

Comment:
String interpolation isn't necessary here. I think you can just reference `field`.